### ISSUE TITLE
Add multiple languages

### ISSUE THREAD

**Issue by PtahSeven** [safing/portmaster#1176]
**What would you like to add or change?**:
Add multiple languages

**Why do you and others need this?**:
I like your product software very much and I am a willing user. Hope to support multiple languages. Chinese traditional, Chinese simplified language

**Comment by dhaavi** [comment]
Hey @PtahSeven, thanks a lot offering your help.

Before we start offering multiple languages we need to add support this in the UI first, so this will sadly take a while until we get there.

Also, this is a duplicate of https://github.com/safing/portmaster/issues/360 - please also comment there about your offer to help so we can keep that all in one place.
